src/test/java/gov/healthit/chpl/aqa/pageObjects/SearchPage.java

```diff
@@ -154,4 +154,24 @@ public static WebElement searchField(final WebDriver driver) {
         return element;
     }
 
+    /**
+     * Returns 'Select All' link on Certification Status' filter.
+     * @param driver WebDriver
+     * @return link element
+     */
+    public static WebElement selectAllStatus(final WebDriver driver) {
+        element = driver.findElement(By.xpath("//*[@id=\"filters\"]/div[2]/div[1]/ul/st-list-multiple/ul/li[1]/a[1]"));
+        return element;
+    }
+
+    /**
+     * Returns item count text on search results.
+     * @param driver WebDriver
+     * @return text element
+     */
+    public static WebElement totalItemCount(final WebDriver driver) {
```

Comment:
In my version of the code there's already a WebElement called "resultCount" that has the same xpath.